lib/models/Place.js

```diff
@@ -0,0 +1,16 @@
+import mongoose from "mongoose";
+
+const { Schema } = mongoose;
+
+const placeSchema = new Schema({
+  lat: { type: Number, required: true },
+  lng: { type: Number, required: true },
+  expirience: { type: String, required: false },
```

Comment:
There is a typo in `experience` here. 

I would advise installing this extension which should provide you with some typo warnings. 

[Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)